src/schema/__tests__/filter_artworks.test.js

```diff
@@ -1,44 +1,35 @@
-import schema from "schema"
 import { runQuery } from "test/utils"
 import { toGlobalId } from "graphql-relay"
 
 describe("Filter Artworks", () => {
+  let rootValue = null
   describe(`Does not pass along the medium param if it is "*"`, () => {
-    const Gene = schema.__get__("Gene")
-    const filterArtworks = Gene.__get__("filterArtworks")
-
     beforeEach(() => {
-      const gravity = sinon.stub()
-      gravity.with = sinon.stub().returns(gravity)
-      // This is the key to the test
-      // the 2nd parameter _should not_ include the mediums option, even though it's included below
-      gravity
-        .withArgs("filter/artworks", {
-          gene_id: "500-1000-ce",
-          aggregations: ["total"],
-          for_sale: true,
-        })
-        .returns(
-          Promise.resolve({
-            hits: [
-              {
-                id: "oseberg-norway-queens-ship",
-                title: "Queen's Ship",
-                artists: [],
-              },
-            ],
-            aggregations: [],
-          })
-        )
-
       const gene = { id: "500-1000-ce", browseable: true, family: "" }
-      Gene.__Rewire__("gravity", sinon.stub().returns(Promise.resolve(gene)))
-      filterArtworks.__Rewire__("gravity", gravity)
-    })
 
-    afterEach(() => {
-      filterArtworks.__ResetDependency__("gravity")
-      Gene.__ResetDependency__("gravity")
+      rootValue = {
+        filterArtworksLoader: sinon
+          .stub()
+          .withArgs("filter/artworks", {
+            gene_id: "500-1000-ce",
+            aggregations: ["total"],
+            for_sale: true,
+          })
```

Comment:
We should slowly but surely get rid of `sinon`, Jest comes with its own mocks and fancy matchers. And less dependencies is always nice.

Reply:
Can't use Jest till we get rid of the babel re-writer, we're getting there on that though

Reply:
I meant we can use `jest.fn()`, which is the only thing that ninon offers, you’re thinking of rewiring.

Reply:
good point

Reply:
Actually I used `jest.fn()` in place of `sinon.stub()` stuff in one place here, I was having trouble with `onCall(0).onCall(1)` things, so instead went with https://github.com/artsy/metaphysics/pull/910/files#diff-5120c49d8b7a7bcf5e854f09c39154b4R10 (for `mockReturnValueOnce` chaining), and yea it worked perfectly.